src/components/notifications/NotificationsLayout/NotificationsLayout.scss

```diff
@@ -0,0 +1,3 @@
+.TargetSelector {
```

Comment:
This will fix https://github.com/WalletConnect/web3inbox/issues/254#issuecomment-1757416146